### PR TITLE
Fix logs retention period & Linux logging issue

### PR DESCRIPTION
**Pull request by ChristopheLarchier**

### CODE REVIEW

File: src/libcommonserver/log/log.cpp [hunk 1]
```diff
@@ -26,15 +26,15 @@
...
-#define LOGGER_APP_PURGE_AFTER_HOURS 24
-
 namespace KDC {
 
+const std::wstring Log::instanceName = L"Main";
+const std::wstring Log::rfName = L"RollingFileAppender";
+const std::wstring Log::rfPattern = L"%D{%Y-%m-%d %H:%M:%S:%q} [%-0.-1p] (%t) %b:%L - %m%n";
+const int Log::rfMaxFileSize = 500;  // MB
+const int Log::rfMaxBackupIdx = 4;   // Max number of backup files
+const int Log::logsPurgeRate = 7;    // days
```
Comment (luc-guyot-infomaniak) [inline src/libcommonserver/log/log.cpp:36]: Should it be a common setting shared across client and server ?

File: src/libcommonserver/log/log.cpp [hunk 2]
```diff
@@ -78,29 +78,29 @@ bool Log::configure(bool useLog, LogLevel logLevel, bool purgeOldLogs) {
     }
 
     // Set purge duration
```
Comment (luc-guyot-infomaniak) [inline src/libcommonserver/log/log.cpp:80]:
```suggestion
    // Set purge rate
```

**Review by luc-guyot-infomaniak — APPROVED** [review]

File: src/libcommonserver/log/log.cpp (hunk 1 above)
Comment (ChristopheLarchier) [reply src/libcommonserver/log/log.cpp]: OK, I moved some shared constants to CommonUtility class

File: src/libcommonserver/log/log.cpp (hunk 2 above)
Comment (ChristopheLarchier) [reply src/libcommonserver/log/log.cpp]: Done

**Pull request merged**